1.10.5/rhel7/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Astronomer <humans@astronomer.io>
 LABEL maintainer="Astronomer <humans@astronomer.io>"
 
 LABEL io.astronomer.docker=true
-LABEL io.astronomer.docker.distro="alpine"
+LABEL io.astronomer.docker.distro="rhel"
```

Comment:
I think this was a copy/paste error from whoever donated the RHEL dockerimage. 

cc @schnie